functions/lib/clients/twitterClient.js

```diff
@@ -0,0 +1,69 @@
+const Twitter = require('twitter')
+
+exports.makeTwitterClient = function makeTwitterClient(consumer_key, consumer_secret, access_token, access_token_secret) {
```

Comment:
That works. Alternatively, since you're not using any instance variables/methods from those methods you could make the "private" methods regular functions in that file (that aren't exported). Then you wouldn't need to `.bind(this)`.